test/tasks_test.rb

```diff
@@ -25,7 +25,7 @@ def capture_stderr
     if ActiveRecord::VERSION::MAJOR >= 4
       ActiveRecord::Tasks::DatabaseTasks.database_configuration = config
       ActiveRecord::Tasks::DatabaseTasks.env = RAILS_ENV
-      ActiveRecord::Tasks::DatabaseTasks.migrations_paths = '/'
```

Comment:
this searches the entire filesystem for migrations on rails 5.1: https://travis-ci.org/zendesk/active_record_shards/jobs/215817999